src/App.tsx

```diff
@@ -9,11 +9,20 @@ const App: React.FunctionComponent = () => {
       flexDir="column"
       justify="center"
       align="center"
-      backgroundColor="blue.700"
+      backgroundColor="blue.900"
     >
-      <Heading fontSize="4xl" color="white">
-        Let's get building...
+
+      <Flex
+      flexDir="column"
+      w="250"
```

Comment:
Use design system token for the width here (e.g: "sm", "lg", "3xl")

Reply:
Smaller widths (like padding) use numbers in the design system, larger widths use "sm", "3xl" etc

Reply:
Task in the comment has been acted upon.